tests/conformance_tests/tests/conformance_runner.rs

```diff
@@ -35,13 +126,23 @@ fn is_valid_file(entry: &DirEntry) -> bool {
         Some(file) => file,
         None => return false,
     };
-    for rx in SKIP_TESTS.iter() {
-        if rx.is_match(file_name) {
-            println!("SKIPPING: {}", file_name);
-            return false;
+
+    if SKIP_MODE {
+        for rx in SKIP_TESTS.iter() {
+            if rx.is_match(file_name) {
+                println!("SKIPPING: {}", file_name);
+                return false;
+            }
+        }
+        return file_name.ends_with(".json");
+    } else {
+        for rx in DO_ONLY.iter() {
+            if rx.is_match(file_name) {
+                return file_name.ends_with(".json");
+            }
         }
```

Comment:
This DO_ONLY logic does not change anything functionally. Since the test would have to be recompiled to run a DO_ONLY, this is no different from just adding a simple `if` check to replace this and then it doesn't have this confusing logic for someone reading or using this next